lib/backend/k8s/conversion.go

```diff
@@ -68,13 +68,13 @@ func (c converter) parsePolicyNameNamespace(name string) (string, error) {
 
 // parsePolicyNameNetworkPolicy extracts the Kubernetes Namespace and NetworkPolicy that backs the given Policy.
 func (c converter) parsePolicyNameNetworkPolicy(name string) (string, string, error) {
-	// Policies backed by NetworkPolicies have form "np.projectcalico.org/<ns_name>.<np_name>"
-	if !strings.HasPrefix(name, "np.projectcalico.org/") {
+	// Policies backed by NetworkPolicies have form "knp.default.<ns_name>.<np_name>"
+	if !strings.HasPrefix(name, "knp.default.") {
 		// This is not backed by a Kubernetes NetworkPolicy.
 		return "", "", fmt.Errorf("Policy %s not backed by a NetworkPolicy", name)
 	}
 
-	splits := strings.SplitN(strings.TrimPrefix(name, "np.projectcalico.org/"), ".", 2)
+	splits := strings.SplitN(strings.TrimPrefix(name, "knp.default."), ".", 2)
```

Comment:
can't we just use `/` as the separator for now since that is not a valid char for the namespace.